src/core/crash-handler.c

```diff
@@ -49,7 +49,7 @@ _noreturn_ static void crash(int sig, siginfo_t *siginfo, void *context) {
 
         if (getpid_cached() != 1)
                 /* Pass this on immediately, if this is not PID 1 */
-                (void) raise(sig);
+                (void) rt_sigqueueinfo(getpid_cached(), sig, siginfo);
```

Comment:
hmm, i hope this works properly for all kinds of signals?

i am pretty sure this shoudl be rt_tgsigqueueinfo() so that this doesn't end up in another thread than the one we are running in here.